test/compilationTests/corion/premium.sol

```diff
@@ -11,6 +11,12 @@ contract thirdPartyPContractAbstract {
 
 contract ptokenDB is tokenDB {}
 
+/**
+*
+* @title Corion Platform Premium Token
+* @author iFA @ Corion Platform
+*
+*/
```

Comment:
You simply moved that on top. That's IMHO not part of this PR (apart from cleaning up maybe?) How did this change come in here? (and others following). Why is this PR containing more changes than the PR is actually intending to?

Reply:
Also this